backend/src/services/episode/episode.service.ts

```diff
@@ -16,6 +17,14 @@ class Episode {
     return this.#episodeRepository.getAll();
   }
 
+  public async getById(id: string): Promise<TEpisode> {
+    const episode = await this.#episodeRepository.getById(id);
+    if (!episode) {
+      throw new HttpError({ status: 404, message: 'Episode does not exist!' });
```

Comment:
```suggestion
      throw new HttpError({ status: 404, message: 'Episode does not exist!' });
```
use http code for status
and create `ErrorMessage` enum in `backend/src/common/enums/app`